openIt/jsonld-contexts/gate.jsonld

```diff
@@ -0,0 +1,7 @@
+{
+  "@context": {
+    "Gate": "https://vocab.egm.io/Gate",
+    "location": "https://vocab.egm.io/location",
+    "name": "https://vocab.egm.io/name"
```

Comment:
for `name`, we usually use `https://schema.org/name`

Reply:
Anyway isn't it already in the core context ?